roxctl/common/printer/tabularprinter_factory.go

```diff
@@ -33,22 +53,23 @@ func NewTabularPrinterFactory(headers []string, rowJSONPathExpression string) *T
 }
 
 // NewTabularPrinterFactoryWithAutoMerge creates new TabularPrinterFactory with the injected default values
-func NewTabularPrinterFactoryWithAutoMerge(headers []string, columnsToMerge []string, rowJSONPathExpression string) *TabularPrinterFactory {
+func NewTabularPrinterFactoryWithAutoMerge() *TabularPrinterFactory {
 	return &TabularPrinterFactory{
 		Merge:                 true,
-		Headers:               headers,
-		RowJSONPathExpression: rowJSONPathExpression,
-		columnsToMerge:        columnsToMerge,
+		Headers:               defaultImageScanHeaders,
+		RowJSONPathExpression: defaultImageScanJSONPathExpression,
+		columnsToMerge:        defaultColumnsToMerge,
 	}
 }
 
 // AddFlags will add all tabular printer specific flags to the cobra.Command
 func (t *TabularPrinterFactory) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().BoolVar(&t.Merge, "merge-output", t.Merge, "Merge duplicate cells in prettified tabular output")
 	cmd.PersistentFlags().StringSliceVar(&t.Headers, "headers", t.Headers, "Headers to print in tabular output")
-	cmd.PersistentFlags().StringVar(&t.RowJSONPathExpression, "row-jsonpath-expressions", t.RowJSONPathExpression,
-		"JSON Path expression to create a row from the JSON object. This leverages gJSON (https://github.com/tidwall/gjson)."+
-			" NOTE: The amount of expressions within the multi-path has to match the amount of provided headers.")
+	err := t.propagateCustomHeaders()
+	if err != nil {
+		log.Errorf("%v", err)
```

Comment:
```suggestion
		cmd.env.Logger().Errorf("%v", err)
```

Reply:
`cobra.Command` has no `env.Logger`, but there's `PrintErrf` which should do the trick 👍

Reply:
What about changing `AddFlags()` to return `error`? The `cliEnvironment` with `Logger()` is passed up stack:
https://github.com/stackrox/stackrox/blob/e50cca861465f1ba7b826d97fba9e743bc9b3d04/roxctl/image/scan/scan.go#L101

Reply:
`PrintErrf` does the trick, indeed.